Python/sysmodule.c

```diff
@@ -2146,6 +2146,13 @@ static PyObject *
 sys__clear_type_cache_impl(PyObject *module)
 /*[clinic end generated code: output=20e48ca54a6f6971 input=127f3e04a8d9b555]*/
 {
+    if (PyErr_WarnEx(PyExc_DeprecationWarning,
+                     "sys._clear_type_cache() is deprecated and scheduled"
+                     " for removal in a future version. Use the more general"
+                     " sys._clear_internal_caches() function instead.",
+                     1) < 0){
+        return NULL;
+    }
```

Comment:
```suggestion
    if (PyErr_WarnEx(PyExc_DeprecationWarning,
                     "sys._clear_type_cache() is deprecated and"
                     " scheduled for removal in a future version."
                     " Use sys._clear_internal_caches() instead.",
                     1) < 0)
    {
        return NULL;
    }
```

Reply:
This makes me wonder whether we could have a C helper for those kind of messages just that would say "XXX is deprecated and scheduled for removal in [...]". "Use YYY instead". I don't know how likely we would use it though but at least we would always have the same message.